tests/test_osmnx.py

```diff
@@ -324,3 +326,33 @@ def test_nominatim():
         response_json = ox.nominatim_request(
                             params = params,
                             type = "transfer")
+
+
+def test_utilities():
+
+    import networkx as nx
+    import pytest
+
+    location_point = (37.791427, -122.410018)
+    G = ox.graph_from_point(location_point, distance=500, distance_type='network')
+
+    def on_no_lanes(u, v):
+        return "under_construction"
+
+    path = [65366915, 65326733, 65326736, 65326738, 65319946, 65319944, 65290756, 65312380, 65319942]
```

Comment:
This could change over time and break the tests if new nodes are digitized on OSM.

Reply:
This test feels a bit unnecessarily complex currently for testing `get_route_edge_attributes`.

Reply:
Do you care of there is no test?

Reply:
We can leave it without a test for now.

Reply:
We can leave it without a test for now.

Reply:
@RunOrVeith if you want to make this update, we can merge the PR after

Reply:
I removed the test case